packages/webdriverio/src/commands/browser/$.js

```diff
@@ -39,7 +39,10 @@ import { elementErrorHandler } from '../../middlewares'
 export default async function $ (selector) {
     const { using, value } = findStrategy(selector)
     const res = await this.findElement(using, value)
+
     const prototype = Object.assign(getWebdriverPrototype(), getWDIOPrototype('element'))
+    prototype.scope = 'element'
```

Comment:
Seems like the following can be used instead?
const prototype = Object.assign(getWebdriverPrototype(), getWDIOPrototype('element'), { scope: 'element' })